wildfly-modules/pom.xml

```diff
@@ -261,6 +222,24 @@
 
    <build>
       <plugins>
+         <plugin>
+            <groupId>org.wildfly.build</groupId>
+            <artifactId>wildfly-server-provisioning-maven-plugin</artifactId>
+            <version>1.2.6.Final</version>
```

Comment:
Nitpick, but could you uitlise the `${version.org.wildfly.build-tools}` property? It's currently on version `1.2.3.Final` but I can't imagine bumping this to `1.2.6.Final` causing any issues.

Reply:
thanks, I missed that this wasn't the first use.
I'll use the property as you suggest, and add a separate ticket to update it (which I include in this PR_